airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -101,7 +105,7 @@ public class ConnectionManagerWorkflowImpl implements ConnectionManagerWorkflow
   private final AutoDisableConnectionActivity autoDisableConnectionActivity =
       Workflow.newActivityStub(AutoDisableConnectionActivity.class, ActivityConfiguration.SHORT_ACTIVITY_OPTIONS);
   private final CheckConnectionActivity checkActivity =
-      Workflow.newActivityStub(CheckConnectionActivity.class, ActivityConfiguration.CHECK_ACTIVITY_OPTIONS);
+      Workflow.newActivityStub(CheckConnectionActivity.class, ActivityConfiguration.SHORT_ACTIVITY_OPTIONS);
```

Comment:
The CheckConnection activity that is launched before a sync workflow was currently using a different set of activity options than the other activities launched by the connection manager workflow. Specifically, it was configured to not have any retry logic, since it was reusing the options from the "check connection" that is launched by the UI.

This change just makes the options used for this activity consistent with the other activities in this workflow, so that they all have the same retry behavior.

Reply:
FYI @evantahler